### ISSUE TITLE
Potentially interested in adding Typescript annotations?

### ISSUE THREAD

**Issue by acorncom** [sandydoo/ember-google-maps#27]
Hey @sandydoo, I've recently started working for another client were we're likely to use your addon there too. That client uses Typescript fairly heavily and might be interested in helping add Typescript annotations to your addon if you're interested. There is a [quest issue to add TypeScript support throughout the Ember.js ecosystem](https://github.com/typed-ember/ember-typings/issues/14) and according to them, it will benefit *everyone* who uses this addon, not just TypeScript users! (See [this discussion for more details](https://github.com/ember-cli/ember-ajax/issues/346)

Would you be interested in either converting the addon to use TypeScript itself, or in adding type definitions? I’d love to help out if so ...

Thanks!

**Comment by sandydoo** [comment]
Yeah, that sounds good @acorncom. Providing type definitions for the public API would be a good start.

**Comment by tdwesten** [comment]
Yeah! This would be great!
